sampuru-ui/src/cases.ts

```diff
@@ -26,11 +36,25 @@ export function casesPage(cases: CaseCard[], project: string): HTMLElement {
   header.innerHTML += ")";
   cardContainer.appendChild(header);
 
+  const internalButton = constructButton("internal-case", "OICR Identifiers", "case-identifier");
+  const externalButton = constructButton("external-case", "External Identifiers", "case-identifier");
+
+  internalButton.onclick = function() {
+    window.location.href = urlConstructor("cases.html", ["cases-project-id", "identifier"], [project, "internal"])
+  }
+
+  externalButton.onclick = function() {
+    window.location.href = urlConstructor("cases.html", ["cases-project-id", "identifier"], [project, "external"])
+  }
+
+  cardContainer.appendChild(internalButton);
+  cardContainer.appendChild(externalButton);
```

Comment:
Should both buttons always be included? I don't think it would be useful and it may be confusing to see the button to change to the view that you're already in